lib/active_merchant/billing/gateways/pay_trace.rb

```diff
@@ -169,19 +169,21 @@ def scrub(transcript)
         transcript.
           gsub(%r((Authorization: Bearer )[a-zA-Z0-9:_]+), '\1[FILTERED]').
           gsub(%r(("credit_card\\?":{\\?"number\\?":\\?")\d+), '\1[FILTERED]').
-          gsub(%r(("cvv\\?":\\?")\d+), '\1[FILTERED]').
+          gsub(%r(("csc\\?":\\?")\d+), '\1[FILTERED]').
           gsub(%r(("username\\?":\\?")\w+@+\w+.+\w+), '\1[FILTERED]').
+          gsub(%r(("username\\?":\\?")\w+), '\1[FILTERED]').
```

Comment:
usernames were only scrubbing if they were email addresses, now it will scrub any username. A test was failing now that the username in our fixtures isn't an email

Reply:
Is username a sensitive value? Core views it as `safe`